Source/WTF/wtf/RunLoop.h

```diff
@@ -131,7 +131,12 @@ class WTF_CAPABILITY("is current") RunLoop final : public RefCountedSerialFuncti
     class TimerBase {
         friend class RunLoop;
     public:
-        WTF_EXPORT_PRIVATE explicit TimerBase(RunLoop&);
+        explicit TimerBase(RunLoop& runLoop)
```

Comment:
Why do we need this? It seems to constructor should suffice? We can implicitly convert a `RunLoop&` to a `Ref<RunLoop>&&` I think